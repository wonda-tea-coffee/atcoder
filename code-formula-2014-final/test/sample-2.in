myon @@c @a @aba@a @@bb bbb @@@@@ @a test  @ space  aaa test @a@a  test@takoyaki
