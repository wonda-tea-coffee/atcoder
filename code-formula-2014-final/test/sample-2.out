a
aba
bb
c
takoyaki
